package.json

```diff
@@ -46,10 +49,10 @@
     "babel-polyfill": "^6.26.0",
     "braintree-web": "^3.62.2",
     "intersection-observer": "^0.10.0",
-    "node-sass": "^4.14.1",
```

Comment:
node-sass is both obsolete and incompatible with modern Node. Replaced with the proper `sass` package